README.md

```diff
@@ -61,10 +61,39 @@ This project is in active development and not ready for production use. However,
 pip install uarray
 ```
 
+or
+
 ```bash
 conda install -c conda-forge -c uarray uarray
 ```
 
+
+## Development
+
+Make a fork of the main [uarray repository](https://github.com/Quansight-Labs/uarray) and clone the fork:
+
+```
+git clone https://github.com/<your-github-username>/uarray
+```
+
+### Install
+
+`uarray` and all development dependencies can be installed via:
+
+```
+pip install -e ".[all]"
+```
+
+Note that uarray supports Python versions >= 3.5.
+
+### Testing
+
+Tests can be run from the main uarray directory as follows:
+
+```
+pytest
+```
+
 ### Releases
```

Comment:
This whole releases section is invalid now. 🙁

Reply:
I'm not sure how releases are currently handled. I can remove this section altogether if you'd like.

Reply:
That'd be nice, I use `twine`. See https://github.com/pydata/sparse/blob/master/release-procedure.md